src/Stream/ReadOnlyFile.php

```diff
@@ -148,6 +148,10 @@ public function close(): void
      */
     public function getHash(): string
     {
+        if ($this->hash) {
+            $this->toctouTest();
+            return $this->hash;
+        }
```

Comment:
Could you write some tests about these changes?

Reply:
There are existing tests that call getHash()

Reply:
Ok. I see that. Thanks for your reply/explanation :).